### ISSUE TITLE
unlock only possible via file

### ISSUE THREAD

**Issue by basz** [AGWA/git-crypt#116]
would it be an idea to be able to encrypt via environment variables?

git-crypt unlock $GITCRYPT_SYMETRIC_KEY

Now that gitlab-ci is able properly secure these, "Secret Variables; These variables will be set to environment by the runner. So you can use them for passwords, secret keys or whatever you want."

Now, I store them as base64 encoded in the var
```
echo $GITCRYPT_SYMETRICKEY_PATH > .git-crypt-base64.key && \ 
openssl base64 -d -in .git-crypt-base64.key -out .git-crypt.key && \
git-crypt unlock .git-crypt.key \
&& rm .git-crypt*.key
```

**Comment by aaronjensen** [comment]
Would this work: 

```bash
git-crypt unlock <(echo $GITCRYPT_SYMETRIC_KEY | base64 --decode)
```

**Comment by aaronjensen** [comment]
@IvanSorokin No, I'm sorry. You'll need to talk to someone who is proficient with powershell, which I am not.
